### DISCUSSION TITLE
parameter: OIDCUserInfoTokenMethod

### DISCUSSION THREAD

**Discussion by mingechen2008** [OpenIDC/mod_auth_openidc#1275] [Q&A]
About parameter OIDCUserInfoTokenMethod, is there any plan to support URI Query Parameter ""access_token"? This is also a spec in RFC 6750.

**Comment by zandbelt** [comment]
https://openid.net/specs/openid-connect-core-1_0.html#UserInfoRequest suggests that that would be invalid per OpenID Connect spec, which supersedes OAuth 2.0 for this request
